### PR TITLE
Added simple health checks

### PR DESCRIPTION
**Pull request by jlecomte**
This will be used as part of our continuous delivery pipeline.
/cc @jamesalley @juandopazo

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

File: Gruntfile.js [hunk 1]
```diff
@@ -44,13 +44,26 @@ module.exports = function (grunt) {
                     }
                 }
             }
+        },
+
+        shell: {
+            health_check: {
+                command: 'mkdir -p artifacts/test/health-check &&' +
```
Comment (juandopazo) [inline Gruntfile.js]: What?!

Comment (juandopazo) [inline Gruntfile.js]: For local testing we don't need to create result files. Can we remove this and simply declare a new script in package.json?

File: Gruntfile.js [hunk 2]
```diff
@@ -44,13 +44,26 @@ module.exports = function (grunt) {
...
     });
 
     // npm tasks.
-    grunt.loadNpmTasks('grunt-broccoli-build');
     grunt.loadNpmTasks('grunt-contrib-clean');
+    grunt.loadNpmTasks('grunt-broccoli-build');
     grunt.loadNpmTasks('grunt-pure-grids');
+    grunt.loadNpmTasks('grunt-shell-spawn');
+
+    grunt.registerTask('build', ['clean', 'broccoli_build', 'clean:tmp']);
```
Comment (juandopazo) [inline Gruntfile.js]: I like this small change!

**Pull request closed without merging**